src/storage-preview/azext_storage_preview/operations/account.py

```diff
@@ -34,7 +35,6 @@ def create_storage_account(cmd, resource_group_name, account_name, sku=None, loc
 
     if NetworkRuleSet and (bypass or default_action):
         if bypass and not default_action:
-            from knack.util import CLIError
             raise CLIError('incorrect usage: --default-action ACTION [--bypass SERVICE ...]')
```

Comment:
Should we consider replacing CLIError with a specific AzCLIError for "Error Handling" project.

Reply:
Currently it could automatically convert to AzCLIError and will consider the migration in next PR.